app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= render 'includes/second_navigation', back_link: 'javascript:history.back()', title: 'Add Category' %>
+
+<section>
+  <div class="container h-100">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+      <div class="col-lg-12 col-xl-11">
+        <div class="card text-black" style="border-radius: 25px;">
+          <div class="card-body p-md-5">
+            <div class="row justify-content-center">
+              <div class="col-md-10 col-lg-6 col-xl-5 order-2 order-lg-1">
+
+                <p class="text-center h1 fw-bold mb-5 mx-1 mx-md-4 mt-4">Record a new transaction</p>
+                <%= form_with model: @transaction, url: transactions_path, class: 'mx-1 mx-md-4' do |f| %>
+                  <div class="d-flex flex-row align-items-center mb-4">
+                    <div class="form-outline flex-fill mb-0">
+                      <%= f.text_field :name, autofocus: true, autocomplete: "name", class: 'form-control', required: true %>
+                      <%= f.label :name, 'Transaction name', class: 'form-label' %>
+                    </div>
+                  </div>
+                  <div class="d-flex flex-row align-items-center mb-4">
+                    <div class="form-outline flex-fill mb-0">
+                      <%= f.number_field :amount, autocomplete: "amount", class: 'form-control', required: true %>
+                      <%= f.label :amount, 'Transaction amount', class: 'form-label' %>
+                    </div>
+                  </div>
+                  <div class="d-flex flex-row align-items-center mb-4">
+                    <div class="form-outline flex-fill mb-0">
+                      <%= f.select :category_id, @categories.pluck(:name, :id), {}, multiple: true, autocomplete: "category_id", class: 'form-control', required: true %>
+                      <%= f.label :amount, 'Select related categories', class: 'form-label' %>
+                    </div>
+                  </div>
+
+                  <div class="d-flex justify-content-between align-items-center mb-3 mb-lg-4">
+                    <%= f.submit "Submit", class: 'btn btn-primary' %>
+                  </div>
+
+                <% end %>
+              </div>
+              <div class="col-md-10 col-lg-6 col-xl-7 d-flex align-items-center order-1 order-lg-2">
+
+                <img src="<%= image_url 'register.svg' %>"
+                     class="img-fluid" alt="Sample image">

```

Comment:
- [ ] Optional: I think at the last line you shoud write `category name` instead of `transaction amount`.
![record](https://user-images.githubusercontent.com/65860678/189086728-257a1bd2-0135-483e-a9c6-f76587327e15.png)